DEPRECATIONS.md

```diff
@@ -0,0 +1,5 @@
+# Deprecated APIs
+
+### DEP0002
```

Comment:
No worries, [`DEP0001` is already in the making](https://github.com/xing/hops/pull/952/files#diff-2967a94b301ca0ca3e404c4f3d47fac3R3).